ryoanji/src/ryoanji/nbody/traversal_cpu.hpp

```diff
@@ -191,38 +187,42 @@ T2 computeGravity(const TreeNodeIndex* childOffsets, const TreeNodeIndex* intern
 #pragma omp parallel for reduction(+ : egravTot)
     for (LocalIndex i = firstTarget; i < lastTarget; i += groupSize)
     {
-        util::array<Vec4<T1>, groupSize> targets, accAndPot;
+        util::array<Vec4<T1>, groupSize> targets, acc;
```

Comment:
This could be renamed "gradients" or "deriv(ative)s" as it is more than just acceleration as you had intended to capture before.

Reply:
reverted back to `accAndPot`